src/frontend/screens/Library/index.tsx

```diff
@@ -407,9 +421,135 @@ export default React.memo(function Library(): JSX.Element {
   }
 
   // select library
-  const libraryToShow = useMemo(() => {
+  const gamesForAlphabetFilter = useMemo(() => {
```

Comment:
this whole function seems to be a complete copy of the `libraryToShow` function which will be a pain in the ass to maintain all this duplicated code, please find a way to re-use the old function instead of copying over all its content